save-cloud-charts/save-cloud/templates/s3-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if not .Values.s3.external -}}
+
+  {{- if eq .Values.profile "dev" -}}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: s3-secrets
+stringData:
+  s3-storage.endpoint: "http://s3:9000"
```

Comment:
this port `9000` can be also moved to values.yaml and re-used

Reply:
Ok will try to do it

Reply:
Done